src/devtools/mixins/keyboard.js

```diff
@@ -1,23 +1,20 @@
-const navMap = {
-  37: 'left',
-  38: 'up',
-  39: 'right',
-  40: 'down'
-}
+export const LEFT = 37
+export const UP = 38
+export const RIGHT = 39
+export const DOWN = 40
+export const S = 83
```

Comment:
Do you also type in Dvorak? 😆
Using Keycodes doesn't work across keyboards for letters

Reply:
Why? I just tried in Dvorak to be sure and it's working fine... :confused:

Reply:
You're completely right. I had recently a problem with keycodes but it's something a bit different, it only happens with the digits row